### DISCUSSION TITLE
Republish release?

### DISCUSSION THREAD

**Discussion by ZebraFlesh** [semantic-release/semantic-release#2260] [Q&A]
I'm in a strange situation where our private Artifactory server apparently went belly up and lost a few artifacts. Is there any way to get semantic-release to republish existing npm artifacts so I can fix Artifactory? I'm guessing I would need to skip publishing the GitHub release + changelog  since those already exist.

**Comment by travi** [comment]
re-running semantic-release would likely not be what you want in that type of situation. if the releases succeeded with semantic-release originally, a tag was added to the git history for each version. your best bet is probably to checkout each tag and run `npm publish` manually
